### ISSUE TITLE
Nexus 6: When speaking into mic the audio level meter only slightly moves

### ISSUE THREAD

**Issue by PaulKinlan** [GoogleChrome/voice-memos#8]
Compare to N5 which will hit the top of the range.  

On the N6 I can barely get it past the 3rd dot, even though when listening to the recording it clips.

**Comment by paullewis** [comment]
Yeah I need to do more work on this. It's very clumsily done atm. The problem is that the lower frequencies always seem to fire, and there's really only a relatively small frequency window that matters. I'll do some playing around, and tweak. Thanks for the heads up.
